module/zcommon/zfeature_common.c

```diff
@@ -760,6 +760,19 @@ zpool_feature_init(void)
 	    ZFEATURE_FLAG_READONLY_COMPAT, ZFEATURE_TYPE_BOOLEAN, NULL,
 	    sfeatures);
 
+	{
+		static const spa_feature_t longname_deps[] = {
+			SPA_FEATURE_EXTENSIBLE_DATASET,
+			SPA_FEATURE_ENABLED_TXG,
```

Comment:
Do we really care at what TXG long names were actually enabled?  What do we do with this knowledge?

Reply:
I'm not sure if there's any reason when this was added. Maybe for audit purpose. Anyway I'll remove it.

Reply:
Thanks. I am not sure what exactly does the extensible dataset, so I just hope it is there for reason and not just copy-paste.